WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Posts/DashboardPostsListCardCell.swift

```diff
@@ -0,0 +1,290 @@
+//
+//  DashboardPostsListCardCell.swift
+//  WordPress
+//
+//  Created by Hassaan El-Garem on 11/04/2022.
+//  Copyright © 2022 WordPress. All rights reserved.
+//
+
+import UIKit
+
+class DashboardPostsListCardCell: UICollectionViewCell, Reusable {
+
+    // MARK: Views
+
+    private var frameView: BlogDashboardCardFrameView?
+    private var ghostableTableView: UITableView?
+    private var errorView: DashboardCardInnerErrorView?
+
+    lazy var tableView: UITableView = {
+        let tableView = PostCardTableView()
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        tableView.isScrollEnabled = false
+        tableView.backgroundColor = nil
+        let postCompactCellNib = PostCompactCell.defaultNib
+        tableView.register(postCompactCellNib, forCellReuseIdentifier: PostCompactCell.defaultReuseID)
+        tableView.separatorStyle = .none
+        return tableView
+    }()
+
+
+    // MARK: Private Variables
+
+    private var viewModel: PostsCardViewModel?
```

Comment:
I did a few tests with an account with 5 sites, and the `PostsCardViewModel` is being retained on a few occasions:

<img width="756" alt="Screen Shot 2022-04-11 at 15 41 52" src="https://user-images.githubusercontent.com/7040243/162807803-c9b0f25c-6eee-4d5a-9aa0-8e8e0d38e675.png">

I think this is something we might want to address because each of those view models has an `NSFetchedResultsController` and its observing posts.

I believe this doesn't affect many users but it's a good improvement anyway. :)

Reply:
I took a look at this and it seems that the max instances of `PostsCardViewModel` retained at anytime is 5.
The collection view always keeps an extra `DashboardPostsListCardCell`, and each cell holds one view model. So the max here is 3 view models.

And there's always 2 instances of `PostsCardViewController` and each hold one view model as well. These however should be removed in the following PR.

Reply:
Ok! I'll check that again in #18351 and think if there's anything we might be able to do, thanks for the explanation!